content/get-involved.ts

```diff
@@ -77,27 +82,42 @@ const getInvolvedContent: GetInvolvedContent = {
     imgsrc: 'illustrations/climbing-guy.svg',
     mobileImgsrc: 'illustrations/checklist1.svg',
   },
-  volunteerPage: {
-    header: 'Help us break down barriers by joining the team.',
-    body: [
-      'You\'ll be a perfect fit if you:',
-    ],
-    facts: [
+  perfectFit: {
+    header: 'You\'ll Be Perfect If...',
```

Comment:
All of the other titles just capitalize the first word, can we align this one please!

Reply:
fixed